test/inductor/test_cpu_repro.py

```diff
@@ -4214,6 +4214,90 @@ def fn(x1, x2, x3):
             n_veckernel = 6 if op is torch.masked.mean else 3
             check_metrics_vec_kernel_count(n_veckernel)
 
+    @requires_vectorization
+    def test_full_bits_lowp(self):
+        from torch._dynamo.utils import detect_fake_mode
+        from torch._inductor.compile_fx import shape_env_from_inputs
+        from torch._inductor.debug import DebugContext
+        from torch._inductor.graph import GraphLowering
+        from torch._inductor.virtualized import V
+        from torch.fx.passes.fake_tensor_prop import FakeTensorProp
+
+        def check_use_full_bits(func, shapes, dtype, mixed, check_vecn):
```

Comment:
Can we use `run_and_get_cpp_code` to simplify the code?

Reply:
Use `run_and_get_cpp_code ` instead.